### DISCUSSION TITLE
How to delete a job using it's jobid

### DISCUSSION THREAD

**Discussion by anirkm** [taskforcesh/bullmq#1832] [Q&A]
There is a way to remove job using it's custom ID ? and also when adding the same jobId i see that it's ignored by default how can i make it so it replaces the old job with the new one ?

**Comment by manast** [comment]
Please take a look at the documentation:
https://api.docs.bullmq.io/classes/Queue.html#remove
https://api.docs.bullmq.io/classes/Job.html#update
